.travis.python.sh

```diff
@@ -15,9 +20,29 @@ else
     sudo apt-get install wget libicu-dev python3-pip python3-setuptools
 fi
 
-CURRDIR=$(pwd)
+PATH=$PATH:$($PYTHON -c "import site; print(site.USER_BASE)")/bin
+
+case $WRAPPER in
+"cython")
+    BUILD_CYTHON="ON"
+    BUILD_PYBIND="OFF"
+    TYPEDEF_POINTS_TO_VECTORS="OFF"
+
+    $PYTHON -m pip install --user -r ./cython/requirements.txt
```

Comment:
Set path to `$GITHUB_WORKSPACE/cython/requirements.txt` to make this script movable?